src/main/webapp/file-edit-button-fragment.xhtml

```diff
@@ -77,6 +77,22 @@
             </h:outputLink>
         </li>
     </ui:fragment>
+    
+    <!-- Single file uningest/reingest -->
+    <ui:fragment rendered="#{isFilePg and dataverseSession.user.superuser and (FilePage.fileMetadata.dataFile.isTabularData() or FilePage.fileMetadata.dataFile.isIngestProblem())}">
+        <li>
+            <p:commandLink update="@form,:messagePanel,:fileForm:fileTitleFragment, :fileForm:topDatasetBlockFragment" action="#{FilePage.uningestFile()}">
+                <h:outputText value="#{bundle['file.uningest']}"/>
+            </p:commandLink>
+        </li>
+    </ui:fragment>
+    <ui:fragment rendered="#{isFilePg and dataverseSession.user.superuser and FilePage.isIngestable()}">
+        <li>
+            <p:commandLink update="@form,:messagePanel" actionListener="#{FilePage.ingestFile()}">
+                <h:outputText value="#{bundle['file.ingest']}"/>
+            </p:commandLink>
+        </li>
+    </ui:fragment>
```

Comment:
Under "Does this PR introduce a user interface change?" I'd be happy to see a screenshot of this.